README.md

```diff
@@ -179,32 +179,39 @@ The default configuration can be read and changed in `config/default.json`.
   "fetcher": {
     "waitForElementsTimeout": "Maximum time (in milliseconds) to wait for elements to be present in the page when fetching document in a headless browser"
   },
-  "notifier": {
-    "sendInBlue": {
+  "notifier": { // Notify specified mailing lists when new versions are recorded
+    "sendInBlue": { // SendInBlue API Key is defined in environment variables, see below
       "administratorsListId": "SendInBlue contacts list ID of administrators",
       "updatesListId": "SendInBlue contacts list ID of persons to notify on document updates",
       "updateTemplateId": "SendInBlue email template ID used for updates notifications"
     }
+  },
+  "logger": { // Logging mechanism to be notified upon error
+    "smtp": {
+      "host": "SMTP server hostname",
+      "username": "User for server authentication" // Password for server authentication is defined in environment variables, see below
+    },
+    "sendMailOnError": { // Can be set to `false` if you do not want to send email on error
+      "to": "The address to send the email to in case of an error",
+      "from": "The address from which to send the email",
+      "sendWarnings": "Boolean. Set to true to also send email in case of warning",
+    }
+  },
+  "tracker": { // Tracking mechanism to create GitHub issues when document content is inaccessible
+    "githubIssues": {
+      "repository": "GitHub repository where to create isssues",
+      "label": "GitHub issues label to attach to a newly created issue. This specific label has to exist in the corresponding repository"
+    }
+  },
+  "dataset": { // Release mechanism to create dataset periodically
+    "publish": "Boolean. Set to true to enable dataset creation"
   }
 }
 ```
 
 The default configuration is merged with (and overridden by) environment-specific configuration that can be specified at startup with the [`NODE_ENV` environment variable](#node-env).
 
```

Comment:
I would add the following as it's not obvious how you can setup your local environment.

Should you wish to modify your local config, we suggest you create a `config/development.json` with overriden values.